src/store/store.ts

```diff
@@ -1216,12 +1252,32 @@ export const useStore = defineStore('store', {
         this.getEntitySnapshot?.businessDocuments?.documentsInfo?.certifiedRules?.key
     },
 
+    // Grab the latest resolution from the entity snapshot.
+    getLatestResolutionForBusiness (): SpecialResolutionIF {
+      // Obtain latest resolution ID. Assumes that the latest resolution is the one to be corrected.
+      const latestResolution = this.getEntitySnapshot.resolutions
+        .reduce((prev, current) => (prev.id > current.id) ? prev : current)
+      return {
+        ...latestResolution,
+        resolutionDate: latestResolution.date
+      }
+    },
+
+    // Only used for correction filings.
+    hasSpecialResolutionResolutionChanged (): boolean {
+      const entitySnapshotResolution = this.getLatestResolutionForBusiness
+      return entitySnapshotResolution.resolution !== this.getSpecialResolution.resolution ||
+             !isEqual(entitySnapshotResolution.signatory, this.getSpecialResolution.signatory) ||
+             entitySnapshotResolution.resolutionDate !== this.getSpecialResolution.resolutionDate ||
+             entitySnapshotResolution.signingDate !== this.getSpecialResolution.signingDate
```

Comment:
It's nice to see all conditions on separate lines. I'd like to see opening and closing parentheses, please and thanks.